archunit/src/main/java/com/tngtech/archunit/core/importer/ImportOption.java

```diff
@@ -76,6 +76,18 @@ public boolean includes(Location location) {
             public boolean includes(Location location) {
                 return doNotIncludeArchives.includes(location);
             }
+        },
+
+        /**
+         * Do not include package-info.class files
```

Comment:
Reading the Javadocs of these classes now I think the best thing would be to write comprehensive Javadoc on the class and then simply `@see ImportOptionClass` here :thinking: Because `Predefined` and the classes themselves (that you would use with JUnit testing support) should give the same information. Unfortunately the given Javadocs are not exactly great examples :joy: We should clean that up at some point and always refer from `Predefined` to the class that then has the full Javadoc...